addons/main/script_mod.hpp

```diff
@@ -1,16 +1,19 @@
-// COMPONENT should be defined in the script_component.hpp and included BEFORE this hpp
-
-#define MAINPREFIX z
-#define PREFIX ace
-
-#define MAJOR 3
-#define MINOR 5
-#define PATCHLVL 1
-#define BUILD 0
-
-#define VERSION MAJOR.MINOR.PATCHLVL.BUILD
-#define VERSION_AR MAJOR,MINOR,PATCHLVL,BUILD
-
-// MINIMAL required version for the Mod. Components can specify others..
-#define REQUIRED_VERSION 1.56
-#define REQUIRED_CBA_VERSION {2,3,1}
+// COMPONENT should be defined in the script_component.hpp and included BEFORE this hpp
+
+#define MAINPREFIX z
+#define PREFIX ace
+
+#define MAJOR 3
+#define MINOR 5
+#define PATCHLVL 1
+#define BUILD 0
+
+#define VERSION MAJOR.MINOR.PATCHLVL.BUILD
+#define VERSION_AR MAJOR,MINOR,PATCHLVL,BUILD
+
+#define WEB_URL "http://ace3mod.com/"
+#define PBO_URL url = WEB_URL;
```

Comment:
I think the standard is to not have the ";" in the macro. Sorry for being picky.